metadata.rb

```diff
@@ -4,8 +4,8 @@
 license          'Apache 2.0'
 description      'Installs/Configures route53'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '1.1.0'
+version          '1.1.1'
```

Comment:
Can you revert the version bump. We have tooling that does this automatically and it'll get tripped up on this.